ospd/command/command.py

```diff
@@ -320,7 +320,12 @@ def handle_xml(self, xml: Element) -> Iterator[bytes]:
 
         vt_details = False if _details == '0' else True
 
-        if vt_id and vt_id not in self._daemon.vts:
+        if (
+            self._daemon.vts
+            and len(self._daemon.vts) > 0
```

Comment:
I really think now the line 325 will never be checked.
```python
if sequence:
```
is the same as
```
if sequence is not None and len(sequence) > 0:
```
Sequence is a list like object. As the almighty [PEP 8 standard](https://www.python.org/dev/peps/pep-0008/) suggests if you are checking for emptiness of a sequence you should just use `If sequence:` instead of `if len(sequence):` or `if len(sequence) > 0:`. BUT I am not 100% sure if our Vts class supports that. So maybe it is better to just use `if len(self._daemon.vts) > 0:` here again as it was originally.

I just wanted to give a hint about the PEP 8 sequence emptiness check recommendation.

Reply:
After a test, it seems our Vts class supports this check. So, solved. Thanks!